src/ensembl/src/content/app/species/state/sidebar/speciesSidebarSlice.ts

```diff
@@ -58,17 +60,23 @@ export type SpeciesSidebarPayload = {
   strain: Strain | null;
 };
 
+type StateForGenome = {
+  payload: SpeciesSidebarPayload | null;
+  isSidebarOpen: boolean;
+};
+
+const initialStateForGenome: StateForGenome = {
+  payload: null,
+  isSidebarOpen: true
+};
+
 type SpeciesPageSidebarState = {
-  isOpen: boolean;
   species: {
-    [genomeId: string]: {
-      payload: SpeciesSidebarPayload | null;
-    };
+    [genomeId: string]: StateForGenome;
   };
 };
 
 const initialState: SpeciesPageSidebarState = {
-  isOpen: true,
   species: {}
```

Comment:
As a side note, why did we decide to have the `species` key in the `SpeciesPageSidebarState`? Are we expecting any other keys at the same level?

Reply:
Good question. I am not sure if we will be having any other keys at the moment. I wonder if we will be having more sections within the sidebar like we have on other apps like bookmarks.

Reply:
In EntityViewer (see PR https://github.com/Ensembl/ensembl-client/pull/440, which we pair-programmed with Jyo), we decided to put bookmarks in their own slice of the state. Sidebar is more for the UI purposes, whereas bookmarks is a more abstract concept, not strictly related to any UI element.

I'm inclined to deleting this species field, but I am leaving the decision to you. We may wait and see if anything else pops up in the sidebar.

Reply:
I have now deleted the key as discussed during the code review.